lib/packagekit-glib2/pk-progress-bar.c

```diff
@@ -172,10 +177,13 @@ pk_progress_bar_pulse_bar (PkProgressBar *self)
 	for (i = 0; i < (gint) (self->priv->size - self->priv->pulse_state.position - 1); i++)
 		g_string_append (str, " ");
 	g_string_append (str, "] ");
-	if (self->priv->percentage >= 0 && self->priv->percentage != PK_PROGRESS_BAR_PERCENTAGE_INVALID)
-		g_string_append_printf (str, "(%i%%)  ", self->priv->percentage);
-	else
+	if (self->priv->percentage >= 0 && self->priv->percentage != PK_PROGRESS_BAR_PERCENTAGE_INVALID) {
+		/* TRANSLATORS: this is a percentage value we use in messages, e.g. "90%. */
+		g_string_append_printf (str, _("(%i%%)"), self->priv->percentage);
+		g_string_append_printf (str, "  ");
+	} else {
 		g_string_append (str, "        ");
```

Comment:
From the MR description it appears that there could be extra spaces in the percentage string depending upon the locale. So, this needs to be done dynamically, I guess.

Reply:
I think the issue stems from the application of `pk_info_enum_to_localised_present` and `pk_info_enum_to_localised_past`. I need to look a bit deeper when I have more time. This particular line should not contribute to the misalignment if I read the code correctly.

Reply:
Is there any chance that translators will insert a lot of spaces in the translated string or make this really long? TBH, the space alignment on stdout is pretty bad already, so it probably wouldn't make much of a difference, but we should at least not make things worse ^^
If translators won't make the % translation long and e.g. just change the ordering, then this patch is fine as-is.

Reply:
The longest variant is `100 %`, which is used in several European countries. I think it should be alright.